content/community-update/community-update/dcp-updates.md

```diff
@@ -6,6 +6,37 @@ title: DCP Updates
 
 # DCP Platform Updates
 
+#### February 28, 2022
```

Comment:
```suggestion
#### March 31, 2022
```

Reply:
Should this be when the release actually went live? I'm assuming yes, which isn't what I originally thought. My bad!

Reply:
I think we've been doing the day the release notes are written, which is generally the same day it's live or close to it.

Reply:
Got it. Thanks!

Reply:
Updated.